napari/layers/vectors/vectors.py

```diff
@@ -436,6 +453,27 @@ def edge_width(self, edge_width: Union[int, float]):
         self.events.edge_width()
         self.refresh()
 
+    @property
+    def vector_style(self) -> str:
+        """Vectors display mode: Determines how vectors are displayed.
+
+        VectorStyle.LINE
+                Displays vectors as rectangular lines.
+            VectorStyle.TRIANGLE
+                Displays vectors as triangles.
+            VectorStyle.ARROW
+                Displays vectors as arrows.
+        """
+        return str(self._vector_style)
+
+    @vector_style.setter
+    def vector_style(self, vector_style: str):
+        old_vector_style = self._vector_style
+        self._vector_style = VectorStyle(vector_style)
+        if self._vector_style != old_vector_style:
```

Comment:
It should be relatively straightforward to add a test for this:

- make a vectors layer
- connect the vector style event to some counter
- change the vector style to the same thing
- check counter hasn't changed
- change the vector style to a new thing
- check the counter has incremented

Does this make sense @jules-vanaret?

Reply:
Yes that is clear ! Should the test be in `_vispy/_tests/test_vispy_vectors_layer.py` ?